tests/dynamic_menu/management/commands/test_load_menu.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+import pathlib
+import tempfile
+from io import StringIO
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import TestCase
+from faker import Faker
+
+from dynamic_menu.models import DynamicMenu
+
+
+class TestLoadMenuCommand(TestCase):
+
+    def setUp(self):
+        self.data = [
+            {
+                'name': 'First Menu',
+                'name_en': 'First Menu',
+                'name_es': 'Primer Menú',
```

Comment:
Is the platform going to be translated to spanish?

Reply:
Yep